Alignment/OfflineValidation/test/BuildFile.xml

```diff
@@ -1,15 +1,37 @@
 <environment>
-  <bin name="testAlignmentOfflineValidation" file="testAlignmentOfflineValidation.cpp">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_all.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="validateAlignments">
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test validateAlignments.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-
-  <bin file="testPVPlotting.cpp">
-    <flags PRE_TEST="testAlignmentOfflineValidation"/>
-    <use name="rootmath"/>
-    <use name="roothistmatrix"/>
-    <use name="rootgraphics"/>
-    <use name="Alignment/OfflineValidation"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="DMRsingleAndmerge">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_DMRsingleAndmerge.sh"/>
+    <use name="FWCore/Utilities"/>
+  </bin>
+  <bin file="testAlignmentOfflineValidation.cpp" name="PVsingleAndmerge">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_PVsingleAndmerge.sh"/>
+    <use name="FWCore/Utilities"/>
+  </bin>
+  <bin file="testAlignmentOfflineValidation.cpp" name="SplitVsingleAndmerge">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_SplitVsingleAndMerge.sh"/>
+    <use name="FWCore/Utilities"/>
+  </bin>
+  <bin file="testAlignmentOfflineValidation.cpp" name="GCPfull">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_GCPfull.sh"/>
+    <use name="FWCore/Utilities"/>
   </bin>
-
+  <!-- <bin file="testAlignmentOfflineValidation.cpp" name="PVsingle"> -->
+  <!--   <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_all.sh"/> -->
+  <!--   <use name="FWCore/Utilities"/> -->
+  <!-- </bin> -->
+  <!-- <bin file="testPVPlotting.cpp"> -->
```

Comment:
please don't remove this test.

Reply:
This one had 'test_all.sh' as a pre-requisite, however that required the old version of validateAlignments.py to be run. Plotting libraries in `testPVPlotting.cpp` are anyway implicitly used in unit test "PVall" so I could see no point in preserving this test.